config/candidates-api.yml

```diff
@@ -0,0 +1,111 @@
+---
+openapi: 3.0.0
+info:
+  version: v1
+  title: Apply candidates API
+  contact:
+    name: DfE
+    email: becomingateacher@digital.education.gov.uk
+  description: |
+    API for candidates from DfE’s Apply for teacher training service.
+servers:
+- description: Sandbox (test environment)
+  url: https://sandbox.apply-for-teacher-training.service.gov.uk/candidates-api
+- description: Production
+  url: https://www.apply-for-teacher-training.service.gov.uk/candidates-api
+paths:
+  "/candidates":
+    get:
+      summary: Get a list of candidates
+      description: This endpoint returns a list of candidates
+      responses:
+        '200':
+          description: A list of candidates
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Candidates"
+  "/candidates/{updated_since}":
+    get:
+      summary: Get a list of candidates updated since
+      parameters:
+        - in: path
```

Comment:
Rather than being a path param, could this be a query param? `/candidates/blah` implies `blah` is an ID, whereas `candidates?updated_since=blah` implies a filter